### PR TITLE
Context View doc

### LINKED ISSUE #799: Add Context View to docs

### PR DESCRIPTION
**Pull request by zardra**
Fixes #799 

Added Context View doc and updated keyboard shortcuts for the new command.

### CODE REVIEW

File: docs/_docs/features/context-view.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+## Definition Preview
+
+When you click on a symbol in the [Editing Area](/docs/editor/basics/#editing-area), the symbol's definition will be highlighted in the Context View panel.
+
+![](/static/images/docs/feature-context-view-highlight.png)
+
+Clicking on the **Open in main editor** button at the bottom of the Context View panel moves the cursor to that definition be it in the current file or a different one.
+
+> Content View currently supports [Hack](/docs/languages/hack), [Python](/docs/languages/python), and [C++](/docs/languages/cpp) files.
```
Comment (JoelMarcey) [inline docs/_docs/features/context-view.md]: s/Content View/Context View

File: docs/_docs/features/context-view.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+<br />
+
+* TOC
+{:toc}
+
+## Toggling
+
+To toggle the Context View panel, you can:
+
+1. Press Cmd-I (Ctrl-I on Linux).
```
Comment (JoelMarcey) [inline docs/_docs/features/context-view.md]: Should the commands be code formatted?

File: docs/_docs/features/context-view.md (hunk 1 above)
Comment (JoelMarcey) [inline docs/_docs/features/context-view.md]: The images seem to show Swift code; but Swift isn't listed as one of the languages above. Should Swift be listed here?

**Review by JoelMarcey — CHANGES_REQUESTED** [review]
I had a few comments on this....

File: docs/_docs/features/context-view.md (hunk 1 above)
Comment (modocache) [inline docs/_docs/features/context-view.md]: +1, although to be clear, the images are of Objective-C.

Comment (JoelMarcey) [inline docs/_docs/features/context-view.md]: Oh, are they? ... oh yeah, I guess they are. Thanks!

Comment (zardra) [inline docs/_docs/features/context-view.md]: Should I specifically list Objective-C as well?

**Comment by zardra** [comment]
How's it look now?

**Comment by JoelMarcey** [comment]
@modocache Is Swift also supported too for Context View? Or just Objective-C?

**Comment by modocache** [comment]
I would be very surprised if Swift was also supported. I don't think it is.

**Review by JoelMarcey — APPROVED** [review]

**Pull request closed without merging**